src/handlers/localstorage/userLists.js

```diff
@@ -0,0 +1,16 @@
+import { DefaultTokenLists } from '../../references';
+import { getGlobal, saveGlobal } from './common';
+
+const USER_LISTS = 'userLists';
+const USER_LISTS_SELECTED_LIST = 'userListsSelectedList';
+
+export const getUserLists = network =>
```

Comment:
you can return null as the default empty state here, or you can remove the fallback DefaultTokenLists logic in redux/userLists loadState